### PR TITLE
🐛 fix null crash in case entity

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

**Comment by alex-wilmer** [comment]
testing case: http://localhost:3000/cases/04dc8ee3-ec93-4b56-a546-f96746f8122f

File: src/packages/@ncigdc/containers/CasePage.js [hunk 1]
```diff
@@ -72,11 +69,9 @@ export const CasePageComponent = ({ node, viewer }: TProps) => (
           query={ctx.query}
           node={node}
           totalFiles={node.files.hits.total}
-          ssmTested={_.get(
-            viewer.explore.cases.hits,
-            "edges[0].node.available_variation_data",
-            []
-          ).includes("ssm")}
+          ssmTested={(viewer.explore.cases.hits.edges[0].node
```
Comment (Jephuff) [inline src/packages/@ncigdc/containers/CasePage.js]: Is it possible edges will be empty at some point? Maybe we should still guard against that?

Comment (alex-wilmer) [inline src/packages/@ncigdc/containers/CasePage.js]: nope because that will fail at the route level

Comment (alex-wilmer) [inline src/packages/@ncigdc/containers/CasePage.js]: well.. that is to say, it's failing currently at the route level because ES returns a 404 style error

Comment (Jephuff) [inline src/packages/@ncigdc/containers/CasePage.js]: makes sense 👍

**Pull request merged**